sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/aio/azure_configuration_client_async.py

```diff
@@ -9,56 +9,78 @@
 from azure.core.tracing.decorator_async import distributed_trace_async
 from azure.core.pipeline.policies.distributed_tracing import DistributedTracingPolicy
 from azure.core.pipeline.transport import AsyncioRequestsTransport
-from azure.core.exceptions import ResourceExistsError, ResourceModifiedError, ResourceNotFoundError
+from azure.core.exceptions import (
+    ResourceExistsError,
+    ResourceModifiedError,
+    ResourceNotFoundError,
+)
 from requests.structures import CaseInsensitiveDict
 from ..utils import (
     get_endpoint_from_connection_string,
     escape_and_tolist,
     prep_update_configuration_setting,
-    quote_etag
+    quote_etag,
 )
 from .._generated.aio import ConfigurationClient
 from .._generated.aio._configuration_async import ConfigurationClientConfiguration
-from ..azure_appconfiguration_requests import AppConfigRequestsCredentialsPolicy, AppConfigConnectionStringCredential
+from ..azure_appconfiguration_requests import (
+    AppConfigRequestsCredentialsPolicy,
+    AppConfigConnectionStringCredential,
+)
 from .._generated.models import ConfigurationSetting
 from .._user_agent import USER_AGENT
 
-class AzureAppConfigurationClient():
+
+class AzureAppConfigurationClient:
     """Represents an client that calls restful API of Azure App Configuration service.
 
-        :param connection_string: Connection String (one of the access keys of the Azure App Configuration resource)
-            used to access the Azure App Configuration.
-        :type connection_string: str
+        :param credentials: An object which can provide secrets for the app configuration service
+        :type credentials: azure.AppConfigConnectionStringCredential
+        :param str base_url: base url of the service
 
-    This is the async version of :class:`azure.data.appconfiguration.ConfigurationClient`
+    This is the async version of :class:`azure.appconfiguration.ConfigurationClient`
 
-    Example
+    """
+    def __init__(self, credentials, base_url, **kwargs):
```

Comment:
`base_url` should come before `credentials` (it should be the first parameter(s))

Reply:
> `base_url` should come before `credentials` (it should be the first parameter(s))

In auto generated code, it puts credentials in front of base_url, should we align this with auto generated code?